pkg/daemon/agent/flexvolume/controller_test.go

```diff
@@ -775,33 +775,35 @@ func TestGetAttachInfoFromMountDir(t *testing.T) {
 	assert.Equal(t, "pvc-123", opts.Image)
 	assert.Equal(t, "pool123", opts.Pool)
 	assert.Equal(t, "storageClass1", opts.StorageClass)
-	assert.Equal(t, "testCluster", opts.ClusterName)
+	assert.Equal(t, "testCluster", opts.ClusterNamespace)
 }
 
-func TestParseClusterName(t *testing.T) {
-	clientset := test.New(3)
+func TestParseClusterNamespace(t *testing.T) {
+	for _, namespaceParameter := range [2]string{"clusterNamespace", "clusterName"} {
```

Comment:
I'd suggest you have a helper method that takes the `namespaceParameter`. Then you can define two different unit tests that call with the different parameter. If there is a failure for one of the cases, it will be easier to tell from the name of the test that fails.